source/server/http/admin.cc

```diff
@@ -801,7 +802,10 @@ Http::ServerConnectionPtr AdminImpl::createCodec(Network::Connection& connection
       new Http::Http1::ServerConnectionImpl(connection, callbacks, Http::Http1Settings())};
 }
 
-bool AdminImpl::createNetworkFilterChain(Network::Connection& connection) {
+bool AdminImpl::createNetworkFilterChain(
+    Network::Connection& connection,
+    const std::vector<Network::NetworkFilterFactoryCb>& filter_factories) {
+  UNREFERENCED_PARAMETER(filter_factories);
```

Comment:
nit: remove UNREFERENCED_PARAMETER, and lave this un-named in the function parameters

Reply:
TIL, thanks!